packages/features/form-builder/FormBuilder.tsx

```diff
@@ -306,8 +305,12 @@ export const FormBuilder = function FormBuilder({
                       {field.label || t(field.defaultLabel || "")}
                     </div>
                     <div className="flex items-center space-x-2">
-                      <Badge variant="gray">{isRequired ? "Required" : "Optional"}</Badge>
-                      {field.hidden ? <Badge variant="gray">Hidden</Badge> : null}
+                      {field.hidden ? (
+                        // Hidden field can't be required, so we don't need to show the Optional badge
+                        <Badge variant="gray">Hidden</Badge>
```

Comment:
Can you add translation for hidden and required?

Reply:
Thanks for pointing out @CarinaWolli. Fixed these and Label and Placeholder as well.